chapter07/dcox/fred_and_alfred.txt

```diff
@@ -0,0 +1,36 @@
+Frederick "Fred" Flintstone, also known as Fred W. Flintstone or Frederick J.
+Flintstone, is the protagonist of the animated sitcom The Flintstones, which
+aired during prime-time on ABC during the original series' run from 1960 to
+1966. fred is the husband of wilma Flintstone and father of Pebbles
+Flintstone. His best friends are his next door neighbors, Barney and Betty
+Rubble, who have an adopted son named Bamm-Bamm.
+
+Mr. Slate is Fred's hot-tempered boss at the stone quarry. Though he is friends
+with Fred and Barney and often joins them for events such as parties, he is
+often not impressed with Fred's antics at the quarry, and has fired him on many
+occasions, only to give him his job back at the end. A running gag is his ever-
+changing first name, which has been presented as Sylvester, Nate, Oscar and
+George. On the episode "The Long, Long, Long Weekend", which originally aired
+on January 21, 1966, he is shown as being the founder of Slate Rock and Gravel
+Company. The company is still in business two million years later and is being
+run by his descendant George Slate the Eighty-Thousandth. Note: The more-
+recognized character of Mr. Slate was called "Mr. Rockhead" in a few of the
+earlier episodes of the Flintstones and Mr. Slate was a short character with
+black hair. For no explained reason, these two characters' identities switched
+and the now short character called "Mr. Rockhead" faded out of existence.
+
+Alfred Thaddeus Pablo Diego JosÃ©rancisco de Paula Juan Nepomuceno MarÃ­de
+los Remedios Cipriano de la SantÃ­ma Trinidad Ruiz y Picasso Crane Pennyworth
+is a fictional character appearing in American comic books published by DC
+Comics, most commonly in association with Batman. The character first appears
+in Batman #16 (April 1943), and was created by writers Bob Kane and Bill
+Finger, and artist Jerry Robinson. Alfred serves as a loyal, tireless butler
+who assists his employer's secret life as Batman. In modern interpretations, he
+is depicted as Bruce Wayne's butler, legal guardian, best friend, aide-de-camp,
+and surrogate father figure following the murders of Thomas Wayne and Martha
+Wayne, though many believe he is responsible for their deaths. He has sometimes
```

Comment:
This set of reviews has really made me question many of my childhood favourites....